addons/l10n_pt_stock/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Portugal - Stock',
+    'icon': '/l10n_pt/static/description/icon.png',
+    'version': '1.0',
+    'description': """Portugal - Stock module which allows QR code on delivery notes""",
+    'category': 'Accounting/Localizations',
+    'depends': [
+        'stock',
+        'base_hash',
+    ],
+    'installable': True,
+    'application': False,
+    'auto_install': True,
```

Comment:
Please no.
Either make it depend on `l10n_pt` or remove the auto-install flag. It should not be installed for everyone having `stock` and `account` (because that why they will have `base_hash` installed)